packages/docs/docs/concepts/progressive-loading/requirements.md

```diff
@@ -0,0 +1,301 @@
+could be interpolated to produce an alternate image. These images are retrieved
+first for fast initial display, followed by retrieving a full resolution image,
+thus producing a progressively better display as more data is loaded.
+
+The DICOM standards committee is currently adding support in DICOM for
+a new encoding method, High Throughput JPEG 2000 `htj2k`. This encoding
+method can be configured to allow for progressive decoding of images.
+That is, if the first N bytes of the image encoding are available they can be
+decoded to a lower resolution or lossy image. The configuration that allows this
+is the HTJ2K Progressive Resolution (HTJ2K RPCL).
+
+The existing JPEG 2000 encoding and the new HTJ2K in the standard also have a
+format specifying a partial resolution endpoint.
+The exact endpoint needs to be specified in the
+JPIP referenced data URL, but is configured statically below as an example only.
+The options data could be used to provide the exact URL required in a future revision.
+
+Finally, some servers can be configured to serve up reduced (partial) resolution
+versions of images on other URL endpoints.
+
+The progressive loading will improve stack image display just given support
+of HTJ2K progressive resolution encoded data, while volumetric data is improved
+in time to first volume for all back ends when not otherwise configured for
+custom load order. However, the support of different types of reduced
+resolution and streaming responses is quite varied between DICOMweb
+implementations. Thus, this guide provides some additional details on how to
+configure various standard and non-standard based options, as well as details
+on how to set that up in the
+[Static DICOMweb](https://github.com/RadicalImaging/Static-DICOMWeb), mostly
+as an example of how it could be done.
+
+## Stack Viewport Streaming Decode
+
+For stack viewports, larger images can be decoding using a streaming method,
+where the HTJ2K RPCL image is received as a stream, and parts of it decoded
+as it is available. This can improve the stack viewing of images quite considerably,
+without any special server requirements other than the support of HTJ2K RPCL
+encoded data.
+
+## Volume Viewport Interleaved Decode
+
+For volumes, the streaming decode of HTJ2K is typically slower than non-streaming
+decoding because the image retrieval of the low resolution data is immediately
+succeeded by the high resolution data, so it consumes the total bandwidth for
+retrieval regardless of handling, and extra decodes require additional time.
+
+There is an alternative low resolution data first retrieval that can be used.
+Interleaving the images applies to any encoding for a volume.
+That is, fetching every Nth image first allows a 1/Nth frequency image to be
+displayed.
+The interleave code then simply replicates the images to the missing
+positions to produce a low resolution in the longitudinal direction.
```

Comment:
again this is in the context of MPR. It is worth mentioning this fact at the start of this h2